kimchi/src/verifier.rs

```diff
@@ -83,6 +83,7 @@ impl<'a, G: KimchiCurve> Context<'a, G> {
                     ForeignFieldMul => Some(self.verifier_index.foreign_field_mul_comm.as_ref()?),
                     Xor16 => Some(self.verifier_index.xor_comm.as_ref()?),
                     Rot64 => Some(self.verifier_index.rot_comm.as_ref()?),
+                    // _ => Some(&self.verifier_index.gate_comms[&t]),
```

Comment:
why is it commented out? there's no more variants? then why the comment?

Reply:
Because it cannot be added until the first new "configurable gate" is added :-)